example/lib/detail_page.dart

```diff
@@ -17,16 +17,17 @@ class DetailPage extends StatelessWidget {
             TextButton(
                 onPressed: () {
                   showDialog(
-                      context: context,
-                      builder: (ctx) => AlertDialog(
-                            title: const Text('Dialog'),
-                            actions: <Widget>[
-                              TextButton(
-                                  onPressed: () => Navigator.pop(context),
-                                  child: const Text('OK'))
-                            ],
-                          ),
-                      useRootNavigator: false); // <===== It's very IMPORTANT!
+                    context: context,
+                    builder: (ctx) => AlertDialog(
+                      title: const Text('Dialog'),
+                      actions: <Widget>[
+                        TextButton(
+                            onPressed: () => Navigator.pop(context),
```

Comment:
Although there are no issues will be raised by using the outer context, we still prefer the correct (inner) context `ctx` when they were called.